### PR TITLE
Added api.ch.gov.uk integration to docker-compose

### PR DESCRIPTION
**Pull request by naufalmirk**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -152,6 +152,25 @@ services:
...
+      - traefik.http.routers.api.ch.gov.uk.rule=Host(`chs-dev`) && PathPrefix(`/api.ch.gov.uk/`)
+    environment:
+      - CUSTOMER_FEEDBACK_EMAIL_ADDRESS=kochab@companieshouse.gov.uk
+      - ELASTIC_SEARCH_VERSION=5
+      - SHARED_MEMORY_PERCENTAGE=25
+      - MAX_MEMORY_USAGE=262144
+      - COOKIE_DOMAIN=chs-dev
+      - COOKIE_SECRET=ChGovUk-XQrbf3sLj2abFxIY2TlapsJ
+      - ELASTIC_SEARCH_ALL_INDEX=primary_search
+      - ELASTIC_SEARCH_PRIMARY_INDEX=primary_search
```
Comment (peadarkelly) [inline docker-compose.yml:168]: hmmm, we'll defos need mongo DB config in here to fetch officers for a company.

File: docker-compose.yml [hunk 2]
```diff
@@ -152,6 +152,25 @@ services:
       - 18222
     networks:
       - chs-dev
+  api.ch.gov.uk:
+    image: 169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/api.ch.gov.uk:latest
+    container_name: api.ch.gov.uk
+    labels:
+      - traefik.http.routers.api.ch.gov.uk.rule=Host(`chs-dev`) && PathPrefix(`/api.ch.gov.uk/`)
```
Comment (peadarkelly) [inline docker-compose.yml:159]: Would the URL to access this service be `http://chs-dev/api.ch.gov.uk/...` ?

Comment (naufalmirk) [reply docker-compose.yml:159]: yes.

Comment (peadarkelly) [reply docker-compose.yml:159]: Should this be it's own host? `http:/api.ch.gov.uk/`? It's not a web UI

Comment (naufalmirk) [reply docker-compose.yml:159]: agreed. I'll fix that.

**Pull request merged**